src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -47,14 +48,18 @@ if (__DEV__) {
         null
     );
 
+    var source = ReactComponentTreeDevtool.getSource(lastDebugID);
+    var formattedSource = source ?
```

Comment:
Is it worth adding `getFormattedSource()` to `ReactComponentTreeDevtool` for convenience?